src/commandHandlers/help.ts

```diff
@@ -21,12 +21,9 @@ export const command = async (arg: [string, string], embed: MessageEmbed) => {
   } else {
     let text = '';
     commandList.forEach((commandItem) => {
-      text = text
-        .concat(`**${COMMAND_PREFIX}${commandItem.triggers[0]}**  `)
-        .concat(`${commandItem.description}\n`)
-        .concat(
-          'Usage:' + '`' + `${COMMAND_PREFIX}${commandItem.usage}\n` + '`'
-        );
+      text = text.concat(
+        '`' + `${COMMAND_PREFIX}${commandItem.triggers[0]}` + '` | '
+      );
```

Comment:
Rather than doing the `forEach` method here, we *could* map the list to a new array to get the string format we want, then `.join(" | ")` to create the `text` value.

Reply:
oh that would be way more neater (learned how to map items in ts 🙂 )

Reply:
@nhcarrigan great suggestion 👍